lib/poker_hands/hand_rankings/royal_flush.rb

```diff
@@ -1,6 +1,6 @@
 class RoyalFlush
   def check(hand)
-    royal_flush_ranks = ['A', 'K', 'Q', 'J', '10']
+    royal_flush_ranks = ['10', '11', '12', '13', '14']
```

Comment:
Is it a flag that these ranks are duplicated between the `Deck` and here?

Reply:
> Is it a flag that these ranks are duplicated between the `Deck` and here?

yes, but I don't think the other ways of checking if the ranks in a poker hand are A,K,Q,J,10 are any more elegant. Maybe I'm not seeing something though.

Reply:
I'm more trying to point out the data duplication, will chat a little IRL.